indico/modules/events/abstracts/forms.py

```diff
@@ -342,6 +349,12 @@ class BulkAbstractJudgmentForm(AbstractJudgmentFormBase):
     def __init__(self, *args, **kwargs):
         self.event = kwargs.pop('event')
         super(BulkAbstractJudgmentForm, self).__init__(*args, **kwargs)
+        self.accepted_track.description = _("The abstracts will be accepted in this track")
+        self.accepted_contrib_type.description = _("The abstracts will be converted into a contribution of this type")
+        self.session.description = _("The generated contributions will be allocated in this session")
```

Comment:
Same with the two fields above as they can be removed too if there are no tracks/types